### DISCUSSION TITLE
Allow fd command name or path to be configured

### DISCUSSION THREAD

**Discussion by jchia** [PatrickF1/fzf.fish#205] [Ideas]
An Ubuntu, `fd` comes in a package called `fd-find` and the binary is named `fdfind` instead of `fd`.

Currently, I fix the problem by editing _fzf_search_directory.fish for the different binary name but it will be helpful if the plugin has an environment variable allowing the binary name or path to be configured.

**Comment by PatrickF1** [comment]
Hi Joshua. Have you tried symlinking fd-find to fd? https://github.com/PatrickF1/fzf.fish/wiki/Troubleshooting#search-directory-feature-does-not-work

**Reply by jchia** [reply]
Thanks for the quick reply.

I haven't tried, but I'm sure the symlink will work.

Different solutions, different tradeoffs.

The symlink is one more file in the system not formally managed by a package manager, so more sysadmin complexity. Additional, some people on Ubuntu (not me) may already have another fd installed (https://packages.ubuntu.com/focal/amd64/fdclone/filelist), probably the reason why Ubuntu names the fd executable differently in the first place, so a symlink with the same name would clash for someone with fdclone installed. (I don't know how popular fdclone is.)

Adding a variable to fzf means more software complexity for fzf.

I don't have a strong opinion since on my machines I'm fine just using a local hack.

**Reply by PatrickF1** [reply]
Got it! You're right--different solutions, different tradeoffs. Unfortunately, I am not currently open to making the fd command configurable in any way because I think this is a rare use case (in most cases, I think people want to use fd-find rather than fdclone).
